### ISSUE TITLE
All recent versions of sugar-date now uninstallable

### ISSUE THREAD

**Issue by TeaSeaLancs** [andrewplummer/Sugar#642]
Since the publishing of sugar-date & sugar-core 2.0.5 an hour ago, all sugar-date versions in the 2.x branch are uninstallable through NPM. The error is as follows:

```
npm ERR! code ENOLOCAL
npm ERR! Could not install from "node_modules/sugar-core/packages/sugar-core" as it does not contain a package.json file.
```

I think there might have been a publishing error with `sugar-core`?

**Comment by andrewplummer** [comment]
Thanks for this! Yes there was a publishing error... Just pushed out a new version and should be fixed now!

**Comment by TeaSeaLancs** [comment]
Cheers for the fast response mate :) If I could make a small suggestion, might it be worth pinning the `sugar-core` dependencies to specific versions rather than using `^2.0.0`? It's more of a pain to maintain but it means that if this kind of thing happens the only version affected is the new version?

**Comment by andrewplummer** [comment]
Yup that sounds like a great idea. I'll leave this ticket open as a reminder!
